server/api_test.go

```diff
@@ -1 +1,60 @@
 package server
+
+import (
+	"context"
+	"testing"
+
+	"golang.org/x/crypto/bcrypt"
+
+	"github.com/coreos/dex/api"
+	"github.com/coreos/dex/storage/memory"
+)
+
+var s = memory.New()
+var serv = NewAPI(s)
+
+func TestCreatePassword(t *testing.T) {
+	ctx := context.Background()
+	p := api.Password{
+		Email:    "test@example.com",
+		Username: "test",
+		UserId:   "test123",
+	}
+	hash, err := bcrypt.GenerateFromPassword([]byte("test1"), 10)
```

Comment:
Generate this value beforehand instead of requiring each test run to compute a bcrypt.

For example, just use this instead:

```
// bcrypt hash of the value "test1" with cost 10
hash := []byte("$2a$10$XVMN/Fid.Ks4CXgzo8fpR.iU1khOMsP5g9xQeXuBm1wXjRX8pjUtO")
```